async_fuse/Cargo.toml

```diff
@@ -36,6 +36,8 @@ smol = { version = "1.2.5" }
 thiserror = "1.0.22"
 utilities = { git = "https://github.com/datenlord/utilities", rev = "4ef408d" }
 common = {path = "../common"}
+lockfree-cuckoohash = {git = "https://github.com/datenlord/lockfree-cuckoohash", rev = "e5849f18"}
+priority-queue = "1.1.0"
```

Comment:
put it in alphabetic order

Reply:
Good point! Guess there's no lint for it.